src/www/ui/api/Controllers/GroupController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/***************************************************************
+ Copyright (C) 2021 Orange
+ Author: Piotr Pszczola <piotr.pszczola@orange.com>
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***************************************************************/
+/**
+ * @file
+ * @brief Controller for user queries
+ */
+
+namespace Fossology\UI\Api\Controllers;
+
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Fossology\UI\Api\Models\Info;
+use Fossology\UI\Api\Models\InfoType;
+use Fossology\UI\Api\Models\Group;
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\UserDao;
+
+/**
+ * @class GroupController
+ * @brief Controller for Group model
+ */
+class GroupController extends RestController
+{
+
+   /**
+   * Get list of Groups
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseInterface $response
+   * @param array $args
+   * @return ResponseInterface
+   */
+  public function getGroups($request, $response, $args)
+  {
+    $userDao = $GLOBALS['container']->get('dao.user');
```

Comment:
You can use `$this->restHelper->getUserDao()` instead of getting it from `$GLOBALS['container']`.

Reply:
Corrected